src/main/java/com/hyperlocal/server/MerchantController.java

```diff
@@ -15,53 +16,51 @@
 import org.springframework.web.bind.annotation.PutMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.server.ResponseStatusException;
+import org.springframework.http.HttpStatus;
 
 @RestController
 public class MerchantController {
 
   private Connection connection;
-
+  private Utilities util;
   public MerchantController() {
     connection = MySQLConnectionBuilder.createConnectionPool(Constants.DATABASE_URL);
-  }
-
-  /**
-   * Updates details of a merchant
-   * @param merchantID The unique ID of the merchant
-   * @param postInputString JSON serialized {@link Merchant} details
-   * @return CompletableFuture of the updated Merchant
-   */
-  @PutMapping("/v1/merchants/{merchantID}")
-  public CompletableFuture<Merchant> updateMerchant(@PathVariable String merchantID, @RequestBody String postInputString) {
-    // TODO: Rewrite method with ID JWT & phone JWT verification
-    JsonObject merchantJson = JsonParser.parseString(postInputString).getAsJsonObject();
-    String merchantName = merchantJson.get("merchantName").getAsString();
-    String merchantPhone = merchantJson.get("merchantPhone").getAsString();
-    return connection
-    .sendPreparedStatement(Constants.MERCHANT_UPDATE_STATEMENT, Arrays.asList(merchantName, merchantPhone, merchantID))
```

Comment:
You can also clean up MERCHANT_UPDATE_STATEMENT

Reply:
Sounds good